yb-voyager/cmd/importDataFileCommand.go

```diff
@@ -100,18 +104,22 @@ func prepareCopyCommands() {
 	log.Infof("preparing copy commands for the tables to import")
 	dataFileDescriptor = datafile.OpenDescriptor(exportDir)
 	for table, filePath := range tableNameVsFilePath {
-		if fileType == datafile.CSV {
+		if fileFormat == datafile.CSV {
 			if hasHeader {
 				df, err := datafile.OpenDataFile(filePath, dataFileDescriptor)
 				if err != nil {
 					utils.ErrExit("opening datafile %q to prepare copy command: %v", err)
 				}
-				copyTableFromCommands[table] = fmt.Sprintf(`COPY %s(%s) FROM STDIN DELIMITER '%c' CSV HEADER`, table, df.GetHeader(), []rune(delimiter)[0])
+				copyTableFromCommands[table] = fmt.Sprintf(`COPY %s(%s) FROM STDIN WITH (FORMAT %s, DELIMITER '%c', ESCAPE '%s', QUOTE '%s', HEADER)`,
+					table, df.GetHeader(), fileFormat, []rune(delimiter)[0], fileOptsMap["escape_char"], fileOptsMap["quote_char"])
 			} else {
-				copyTableFromCommands[table] = fmt.Sprintf(`COPY %s FROM STDIN DELIMITER '%c' CSV`, table, []rune(delimiter)[0])
+				copyTableFromCommands[table] = fmt.Sprintf(`COPY %s FROM STDIN WITH (FORMAT %s, DELIMITER '%c', ESCAPE '%s', QUOTE '%s')`,
+					table, fileFormat, []rune(delimiter)[0], fileOptsMap["escape_char"], fileOptsMap["quote_char"])
 			}
+		} else if fileFormat == datafile.TEXT {
+			copyTableFromCommands[table] = fmt.Sprintf(`COPY %s FROM STDIN WITH (FORMAT %s, DELIMITER '%c')`, table, fileFormat, []rune(delimiter)[0])
 		} else {
-			panic(fmt.Sprintf("File Type %q not implemented\n", fileType))
+			panic(fmt.Sprintf("File Type %q not implemented\n", fileFormat))
```

Comment:
-- `DataFile` interface should have a method named `GetCopyCommand() string`.
-- "file options" must be a part of `datafile.Descriptor`.